cbor.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Contributors to the Veraison project.
```

Comment:
if this is a new file, then perhaps the year should be 2021?

Reply:
ok